### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by ArshErgon**
Fixed some typos inside readme.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -70,14 +70,14 @@ is to use community discussion to improve proposals and assist authors.
...
-If you have ideas for Carbon, we encourage you to discuss it with the community,
+If you have ideas for Carbon, we encourage you to discuss them with the community,
 and potentially prepare a proposal for it. Ultimately, any changes or
 improvements to Carbon will need to turn into a proposal and go through our
 [evolution process](docs/project/evolution.md).
 
 If you do start working on a proposal, keep in mind that this requires a time
 investment to discuss the idea with the community, get it reviewed, and
-eventually implemented. A good starting point is to read through the
+eventually implemented it. A good starting point is to read through the
```
Comment (fowles) [inline CONTRIBUTING.md]: I don't agree with this edit.  I actually think the error is the `,` before the and.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -88,10 +88,9 @@ and are very interested in folks joining in to help us with it.
 
 ### Help address bugs
 
-As Carbon's design and eventually implementation begin to take shape, we'll
+As Carbon's design and eventually implementation begins to take shape, we'll
 inevitably end up with plenty of bugs. Helping us triage, analyze, and address
-them is always a great way to get involved. When we have the bug tracker(s) set
-up for this, we'll update this section with ideas of how to find these and get
+them is always a great way to get involved. When we have the bug tracker(s) set up for this, we'll update this section with ideas on how to find these and get
```
Comment (fowles) [inline CONTRIBUTING.md]: please run prettifier on this

File: README.md [hunk 3]
```diff
@@ -179,8 +179,8 @@ https://drive.google.com/drive/folders/1-rsUjiya7dSZ87L8kpZmu3MZghRVxzLA
      alt="A snippet of converted Carbon code. Follow the link to read it.">
 </a>
 
-You can call Carbon from C++ without overhead and the other way around. This
-means you migrate a single C++ library to Carbon within an application, or write
+You can call Carbon from C++ without the overhead and the other way around. This
```
Comment (fowles) [inline README.md]: I think the original was correct about no having "the" here

File: README.md [hunk 4]
```diff
@@ -179,8 +179,8 @@ https://drive.google.com/drive/folders/1-rsUjiya7dSZ87L8kpZmu3MZghRVxzLA
...
+means you migrate a single C++ library to Carbon within an application or write
```
Comment (fowles) [inline README.md]: missing word:  "This means you can migrate ..."

**Comment by fowles** [comment]
You will need to sign the cla if you wish to contribute this.

**Review by chandlerc — CHANGES_REQUESTED** [review]
(marking as needing changes, both for the review comments but more importantly for CLA)

File: CONTRIBUTING.md (hunk 2 above)
Comment (ArshErgon) [reply CONTRIBUTING.md]: like how? is it an extension?

Comment (fowles) [reply CONTRIBUTING.md]: https://prettier.io/ is the tool we use.  The exact how depends on what system you are running on.

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -47,10 +47,10 @@ communication. It applies to all of your communication and conduct in these
 spaces, including emails, chats, things you say, slides, videos, posters, signs,
 or even t-shirts you display in these spaces.
 
-All community members should help support our standards of acceptable behavior.
-Everyone is encouraged to speak up in response to any behavior that they deem
-inappropriate, threatening, offensive, or harmful. If you believe someone is
-violating the code of conduct, please report it to the
+- All community members should help support our standards of acceptable behavior.
+- Everyone is encouraged to speak up in response to any behavior that they deem
```
Comment (fowles) [inline CODE_OF_CONDUCT.md]: I see some stray `-` got included here

File: CONTRIBUTING.md (hunk 2 above)
Comment (ArshErgon) [reply CONTRIBUTING.md]: I did please do a review.

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -75,11 +75,11 @@ spaces:
 -   **Be considerate.** Your work will be used by other people, and you in turn
     will depend on the work of others. Any decision you make will affect users
     and colleagues, and you should take those consequences into account.
-    Remember that we’re a world-wide community, so you might not be
+    Remember that we’re a worldwide community, so you might not be
     communicating in someone else’s primary language.
 
 -   **Be kind.** Not all of us will agree all the time, but disagreement is no
-    excuse for poor behavior and hurtful words. We might all experience some
+    excuse for poor behaviour and hurtful words. We might all experience some
```
Comment (fowles) [inline CODE_OF_CONDUCT.md]: I don't know that we want to start fighting over american vs british spellings.  I would prefer to leave as is

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -191,8 +191,7 @@ If the violation occurs at an event and requires immediate attention, you can
 also reach out to any of the event organizers or staff. Event organizers and
 staff will be prepared to handle the incident and be able to help. If you cannot
 find one of the organizers, the venue staff can locate one for you. Specific
-event information will include detailed contact information for that event. In
-person reports will still be kept confidential exactly as above, but also feel
+event information will include detailed contact information for that event. In-person reports will still be kept confidential exactly as above, but also feel
```
Comment (fowles) [inline CODE_OF_CONDUCT.md]: formatting

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (ArshErgon) [reply CODE_OF_CONDUCT.md]: okay, sure. I was thinking the same, just wanted to be a part of Carbon that's why I did that.

File: CODE_OF_CONDUCT.md (hunk 7 above)
Comment (ArshErgon) [reply CODE_OF_CONDUCT.md]: I'm sorry, again re-format it.

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -75,11 +76,11 @@ spaces:
...
     will depend on the work of others. Any decision you make will affect users
     and colleagues, and you should take those consequences into account.
-    Remember that we’re a world-wide community, so you might not be
-    communicating in someone else’s primary language.
+    Remember that we’re a worldwide community, so you might not be communicating
+    in someone else’s primary language.
 
 -   **Be kind.** Not all of us will agree all the time, but disagreement is no
-    excuse for poor behavior and hurtful words. We might all experience some
+    excuse for poor behaviour and hurtful words. We might all experience some
```
Comment (fowles) [inline CODE_OF_CONDUCT.md]: I am still seeing spelling changes for "behavior".  Did you miss a `git stash pop` or something?

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (ArshErgon) [reply CODE_OF_CONDUCT.md]: Is it gonna make any difference? my preview window shows it alright?

File: README.md [hunk 9]
```diff
@@ -231,16 +231,16 @@ and with a smooth evolutionary path.
...
+challenge for C++ and something a successor language needs to address. Our
+initial priority and focus are on immediately addressing important, low-hanging
 fruit in the safety space:
 
 -   Tracking uninitialized states better, increased enforcement of
     initialization, and systematically providing hardening against
     initialization bugs when desired.
 -   Designing fundamental APIs and idioms to support dynamic bounds checks in
-    debug and hardened builds.
+    debugging and hardened builds.
```
Comment (fowles) [inline README.md]: "debug builds" are a term of art, so should remain as such

**Review by fowles — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (ArshErgon) [reply README.md]: sure.

File: CODE_OF_CONDUCT.md (hunk 8 above)
Comment (ArshErgon) [reply CODE_OF_CONDUCT.md]: should I change it back to 'behavior'? I change 'behavior' to 'behaviour' on all occurrences.

Comment (fowles) [reply CODE_OF_CONDUCT.md]: Yes please.

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (fowles) [reply CODE_OF_CONDUCT.md]: after your most recent formatting this is fine.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jonmeow) [reply CONTRIBUTING.md]: I think it's just a clumsy sentence (*that I assume I wrote). Probably a better phrasing would be:

...investment to get an idea discussed with the community, reviewed, and eventually implemented.

Comment (fowles) [reply CONTRIBUTING.md]: I like that phrasing better.  Do you mind using the one suggested by jonmeow?

**Review by fowles — COMMENTED** [review]
I think we are pretty close!  Thanks for sticking with it.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ArshErgon) [reply CONTRIBUTING.md]: sure, done.

**Review by fowles — APPROVED** [review]

**Comment by jonmeow** [comment]
Thank you for your patience and taking comments! I want to note that pre-commit is currently failing with a formatting issue:
https://github.com/carbon-language/carbon-lang/runs/7450985610?check_suite_focus=true

```
-investment to get an idea discussed with the community, reviewed, and
-eventually implemented. A good starting point is to read through the
+investment to get an idea discussed with the community, reviewed, and eventually
+implemented. A good starting point is to read through the
```

Given fowles has approved, I'm pretty sure chandlerc will be okay with this.

File: README.md (hunk 4 above)
Comment (chandlerc) [inline README.md]: Suggested fixes to the above:

```suggestion
You can call Carbon from C++ without overhead and the other way around. This
means you can migrate a single C++ library to Carbon within an application or write
```

File: README.md [hunk 10]
```diff
@@ -231,9 +231,9 @@ and with a smooth evolutionary path.
 ## Memory safety
 
 Safety, and especially
-[memory safety](https://en.wikipedia.org/wiki/Memory_safety), remain key
-challenges for C++ and something a successor language needs to address. Our
-initial priority and focus is on immediately addressing important, low-hanging
+[memory safety](https://en.wikipedia.org/wiki/Memory_safety), remains a key
+challenge for C++ and something a successor language needs to address. Our
+initial priority and focus are on immediately addressing important, low-hanging
```
Comment (chandlerc) [inline README.md]: I find the "is" to "are" change here surprising, but my grasp of this part of the English grammar is quite weak so I'll trust others on this.

**Review by chandlerc — CHANGES_REQUESTED** [review]
Nice!

Suggesting a change below to address a couple of open comments, but once that is done, this should be good to go.

File: README.md (hunk 10 above)
Comment (ArshErgon) [reply README.md]: so should I change it back to 'is'?

Comment (chandlerc) [reply README.md]: No, I'll trust others' review here.

**Review by chandlerc — COMMENTED** [review]
I think this just needs formatting fixes from [`pre-commit`](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/contribution_tools.md#pre-commit)

You can see the GitHub action showing the issue here:
https://github.com/carbon-language/carbon-lang/runs/7479987049?check_suite_focus=true

You should be able to reproduce and automatically fix by getting `pre-commit` working as suggested in these docs.

File: README.md (hunk 10 above)
Comment (ArshErgon) [reply README.md]: that pre-commit got failed at 'prettier', what does it mean? I should format it with prettier on locally?
`- hook id: prettier
- files were modified by this hook`

**Pull request closed without merging**